kea_exporter/base_exporter.py

```diff
@@ -499,8 +499,8 @@ def parse_metrics(self, dhcp_version, arguments, subnets):
                         if f"{subnet_id}-{pool_index}" not in self.subnet_missing_info_sent.get(dhcp_version, []):
                             self.subnet_missing_info_sent.get(dhcp_version, []).append(f"{subnet_id}-{pool_index}")
                             click.echo(
-                                "Ignoring metric because subnet vanished from configuration:",
-                                f"\tdhcp_version: {dhcp_version.name}, subnet_id: {subnet_id}, pool_idx: {pool_index}",
+                                "Ignoring metric because subnet vanished from configuration: "
+                                f"dhcp_version: {dhcp_version.name}, subnet_id: {subnet_id}, pool_idx: {pool_index}",
```

Comment:
And here.

Reply:
Nice catch

Reply:
Are you going to update the commit? :wink:

Reply:
I don't see the button where I can accept the changes, I did for the last change but I cannot seem to click this one.

Reply:
I did not add a suggestion like last time.

```suggestion
                                f"{dhcp_version.name=}, {subnet_id=}, {pool_index=}",
```